openapi.yaml

```diff
@@ -527,6 +550,11 @@ components:
             additionalProperties:
               type: integer
               format: int32
+    JSONObject:
+      type: object
+      properties:
```

Comment:
As mentioned before, this file is auto-generated. It might be related to some properties owned by JSONObject. My understanding is that "command" is a string field under requestBody. (Not sure but correct me if I'm wrong)
<img width="1408" alt="Screen Shot 2022-08-30 at 11 40 24 AM" src="https://user-images.githubusercontent.com/60716841/187517120-8356e88f-b30a-405a-9c0f-0e69ed4cb905.png">